src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,62 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.ArrayList;
+import java.util.List;
+
 public class WorkWithFile {
+    private List<String> values;
+    private List<String> reports = new ArrayList<>();
 
     public void getStatistic(String fromFileName, String toFileName) {
+        reading(fromFileName);
+        calculating();
+        writing(toFileName);
+    }
+
+    private void reading(String fromFileName) {
```

Comment:
```suggestion
    private void read(String fromFileName) {
```
same for other method names